dennis/minisix.py

```diff
@@ -10,5 +10,12 @@
 
 else:
     textclass = str
-    from html.parser import HTMLParser, HTMLParseError  # noqa
+    from html.parser import HTMLParser  # noqa
     from itertools import zip_longest as izip_longest  # noqa
+    try:
+        from html.parser import HTMLParseError  # noqa
+    except:
```

Comment:
:+1:

I'm happy to revise patches during review.
From what I have ready, the better solution is to avoid strict mode, which I believe means that this error wont be raised on Python 2.7.  But I didnt have time to read the Python source to 100% verify that fact, or determine the conditions necessary to avoid this exception occurring.